Source/Paths/SvgPath.cs

```diff
@@ -32,66 +28,55 @@ public SvgPathSegmentList PathData
         [SvgAttribute("pathLength")]
         public float PathLength
         {
-            get { return this.Attributes.GetAttribute<float>("pathLength"); }
-            set { this.Attributes["pathLength"] = value; }
+            get { return GetAttribute<float>("pathLength", false); }
+            set { Attributes["pathLength"] = value; }
         }
 
         /// <summary>
         /// Gets the <see cref="GraphicsPath"/> for this element.
         /// </summary>
         public override GraphicsPath Path(ISvgRenderer renderer)
         {
-            if (this._path == null || this.IsPathDirty)
+            if (_path == null || IsPathDirty)
             {
-                this._path = new GraphicsPath();
+                _path = new GraphicsPath();
 
-                foreach (var segment in this.PathData)
+                if (PathData != null)
                 {
-                    segment.AddToPath(_path);
-                }
+                    foreach (var segment in PathData)
+                        segment.AddToPath(_path);
 
-                if (_path.PointCount == 0)
-                {
-                    if (this.PathData.Count > 0)
+                    if (_path.PointCount == 0)
                     {
-                        // special case with one move command only, see #223
-                        var segment = this.PathData.Last;
-                        _path.AddLine(segment.End, segment.End);
-                        this.Fill = SvgPaintServer.None;
-                    }
-                    else
-                    {
-                        _path = null;
+                        if (PathData.Count > 0)
+                        {
+                            // special case with one move command only, see #223
+                            var segment = PathData.Last;
+                            _path.AddLine(segment.End, segment.End);
+                            Fill = SvgPaintServer.None;
+                        }
+                        else
+                            _path = null;

```

Comment:
@mrbean-bremen 

Is this needed ?

I think _path should be kept.
Because this process is repeated next time.

Reply:
This is a fix for an odd case that will happen very rarely, so it should not matter that much.